app.py

```
@@ -26,27 +26,31 @@
+                os.makedirs(save_dir, exist_ok=True)
+                for photo in photos:
+                    save_path = os.path.join(save_dir, photo.name)
+                    with open(save_path, "wb") as f:
+                        f.write(photo.getbuffer())
+                st.success(f"Saved {len(photos)} file(s) to: {save_dir}")
+
+                if st.button("Run"):
+                    try:
+                        fill_form_from_excel(data, username, password)
+                        st.success("Form filling process completed successfully!")
+                    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)